db/migrate/20200217165218_create_metrics.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMetrics < ActiveRecord::Migration[6.0]
+  def change
+    create_table :metrics do |t|
+      t.string :entity_key,         null: false
```

Comment:
should we add indexes for `entity_key`, `metric_key` and `value_timestamp`?

Reply:
I am not sure about that yet.

I don't have a clear idea yet of how is this table to be accessed in the actual use cases. It is possible that it might be joined by its primary key alone.

As this table is meant to have a lot of records and each index might add a significant space overhead to the table structure I would like to postpone the creation of indices until the use cases are implemented. Would that be possible?

Reply:
Sure